bycycle/group/features.py

```diff
@@ -40,9 +40,9 @@ def compute_features_2d(sigs, fs, f_range, compute_features_kwargs=None, axis=0,
     return_samples : bool, optional, default: True
         Whether to return a dataframe of cyclepoint sample indices.
     n_jobs : int, optional, default: -1
-        The number of jobs, one per cpu, to compute features in parallel.
-    progress: {None, 'tqdm', 'tqdm.notebook'}
-        Specify whether to display a progress bar. Use 'tqdm' if installed.
+        The number of jobs, one per CPU, to compute features in parallel.
```

Comment:
Maybe "one per CPU" should be removed. Some hardware is hyperthreaded to allow for 2 jobs per CPU.

Reply:
Yeh, that's a good point! I think it makes more sense to refer to number of jobs, without stating / assuming anything about CPUs.